Products/ZenModel/migrate/switchLoggingLevel.py

```diff
@@ -0,0 +1,46 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2015, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+__doc__ = """
+Switch default logging level to WARN
+"""
+import logging
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import yaml
+import servicemigration as sm
+sm.require("1.0.0")
+
+
+class SwitchLoggingLevel(Migrate.Step):
+
+    version = Migrate.Version(5, 0, 8)
+
+    def cutover(self, dmd):
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        cqs = filter(lambda s: s.name == 'CentralQuery', ctx.services)
+        if len(cqs) != 1:
+            log.info("Found %i services named 'CentralQuery'; skipping." % len(cqs))
```

Comment:
Please use this form of calling the log methods:
```
    log.info("Found %i services named 'CentralQuery'; skipping.", len(cqs))
```
Note that the '%' string interpolation operator is not used.